docs/README.md

```diff
@@ -15,9 +15,9 @@
     <a href="https://www.vidurjobs.xyz"><strong>Learn more »</strong></a>
     <br />
     <br />
-    <a href="https://discord.gg/Pt2W9Sx7ap">Discord</a>
+    <a href="https://discord.com/invite/cRaukv9dJ2">Discord</a>
```

Comment:
revert this. let @devshahofficial add the official link

Reply:
@amandesai01 okay, reverting this